packages/components/src/List/ListToVirtualizedList/ListToVirtualizedList.component.js

```diff
@@ -62,6 +57,11 @@ function ListToVirtualizedList(props) {
 						columnData: column.data,
 					});
 				}
+				if (titleProps && column.key === titleProps.key) {
```

Comment:
We have 3 cases here : 
* supposed action (for retro compatibility) --> `CellActions`
* explicit type --> the specified cell type
* titleProps defined --> `CellTitle` with props

Those are not compatible with each other. So we have to define an order and use `if/else` instead of `if` x3
I would suggest : titleProps - actions - type (as before). Type is the generic case, so it falls at the end.

Reply:
Hi jimmy,
thanks for explaining! I've updated according your comment.